### ISSUE TITLE
lr scheduler not optimizing params

### ISSUE THREAD

**Issue by jpchen** [uber/pyro#1242]
`svi.step()` is not updating parameters with each optimizer.  see #1241

**Comment by jpchen** [comment]
ahh the pytorch scheduler only updates the scheduler and does not take an optimization step. this was my mistake - i had assumed pytorch was documented correctly and only tested the hyperparam updates not the optimization. sorry for the inconvenience.
